crates/sui-indexer/migrations_v2/2023-08-19-044044_checkpoints/up.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE checkpoints
+(
+    sequence_number                     bigint       PRIMARY KEY,
+    checkpoint_digest                   bytea        NOT NULL,
+    epoch                               bigint       NOT NULL,
+    network_total_transactions          bigint       NOT NULL,
+    previous_checkpoint_digest          bytea,
+    end_of_epoch                        boolean      NOT NULL,
+    tx_digests                          bytea[]      NOT NULL,
+    timestamp_ms                        BIGINT       NOT NULL,
+    -- derived from GasCostSummary
+    total_gas_cost                      BIGINT       NOT NULL,
+    computation_cost                    BIGINT       NOT NULL,
+    storage_cost                        BIGINT       NOT NULL,
+    storage_rebate                      BIGINT       NOT NULL,
+    non_refundable_storage_fee          BIGINT       NOT NULL,
+    checkpoint_commitments              bytea        NOT NULL,
```

Comment:
We currently have two kinds of commitment (previous checkpoint digest and live object set digest) -- how will they be represented in the byte array? (i.e. what's the encoding)?

Reply:
most of the bytes columns are bcs, including live object set. Previous checkpoint digest is byte array